src/services/tech-radar/file.ts

```diff
@@ -0,0 +1,16 @@
+import * as fs from "fs";
+
+export class FileTechRadarClient implements TechRadarClient {
+  constructor(private path: string) {}
+
+  getTechRadarTechs(): Promise<RadarTech[]> {
+    const jsonString = fs.readFileSync(this.path, "utf8");
```

Comment:
instead of keeping the type of the value on the variable name, what about naming just `json` / `content` / `data`?

Reply:
sounds good